### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by joncol**
We made the repository public, so adding a license is probably a good idea.

### CODE REVIEW

File: scrive-prelude.cabal [hunk 1]
```diff
@@ -8,9 +8,9 @@ stability:     Experimental
 category:      Web
 synopsis:      Prelude for scrive
 description:
-  This is scrive's common collection of function import we use the most.
+  This is Scrive's common collection of imports and helper functions.
 
-license:       BSD3
+license:       BSD-3-Clause
```
Comment (theophile-scrive) [inline scrive-prelude.cabal:13]: Has this been sanctioned by Tage btw?

Comment (joncol) [reply scrive-prelude.cabal:13]: Not Tage, but we discussed it during the architecture meeting, and our other public repositories use BSD3, afaik.

**Review by arybczak — APPROVED** [review]

File: scrive-prelude.cabal (hunk 1 above)
Comment (matobet) [reply scrive-prelude.cabal:13]: BSD :+1: :100:

**Review by matobet — APPROVED** [review]

**Review by kutyel — APPROVED** [review]
Let's do this! :shipit: 🙌🏻

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Copyright (c) 2013-2023, Scrive
```
Comment (kutyel) [inline LICENSE:1]:
```suggestion
Copyright © 2013-2023, Scrive
```

Comment (arybczak) [reply LICENSE:1]: This is a standard LICENCE file auto-generated by Cabal, pleas just leave it as-is.

**Pull request merged**